app/views/works/index.html.erb

```diff
@@ -33,7 +33,7 @@
         <% if @tag && logged_in? && !@collection %>
           <li><%= render 'favorite_tags/form', current_user: @current_user, favorite_tag: @favorite_tag %></li>
         <% end %>
-        <% if @tag && !@collection && (%w(Fandom Character Relationship).include?(@tag.type.to_s) || @tag.name == "F/F") %>
+        <% if @tag && !@collection && !params[:work_search] && (%w(Fandom Character Relationship).include?(@tag.type.to_s) || @tag.name == "F/F") %>
```

Comment:
After ES6, we also need to check for `params[:include_work_search]` and `params[:exclude_work_search]`, similar to #3550.